### PR TITLE
Add accordion widget

### PR DESCRIPTION
**Pull request by clintmoss**
Added accordion widget, testing area, and toggle buttons.

### CODE REVIEW

File: docs/components/accordion.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      accordionItems.forEach((accordionItem) => {
+
+        const accordionHeading: HTMLHeadingElement | null = accordionItem.querySelector("accordion-item>h3, accordion-item>h4");
+
+        // Add classes and attributes to the accordion heading and content
+        if (accordionHeading) {
+          accordionHeading.setAttribute("tabindex", "0");
+          accordionHeading.classList.add("toggle-btn");
+          accordionHeading.classList.add("accordion-title");
+          accordionHeading.classList.add("arrow-down");
```
Comment (franreyn) [inline docs/components/accordion.md:168]: I'm not seeing the arrow change from right to bottom. What are your thoughts on adding the functionality now?

File: src/ts/modules/toggleButtons.ts
Comment (franreyn) [inline src/ts/modules/toggleButtons.ts:1]: Can we convert this to a hook? Can you think of any times we may need it?

**Review by franreyn — CHANGES_REQUESTED** [review]
I have a few notes inline. Thank you.

File: docs/components/accordion.md (hunk 1 above)
Comment (clintmoss) [reply docs/components/accordion.md:168]: Yeah, I've added the needed ts code.

File: src/ts/modules/toggleButtons.ts
Comment (clintmoss) [reply src/ts/modules/toggleButtons.ts:1]: toggleButtons is used for the accordion widget and the footer, I don't believe we use it for anything else. 

Should we still rewrite it as a reusable function?

**Review by franreyn — APPROVED** [review]
Thank you

**Pull request merged**